4.6.2.16/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:wheezy
+
+MAINTAINER Jo Shields <jo.shields@xamarin.com>
+
+#based on dockerfile by Michael Friis <friism@gmail.com>
+
+ENV MONO_VERSION 4.6.2.16
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
@dominikhahn where did you get this from? It's not recommended by Docker and I had to remove it.